testscommon/state/accountWrapperMock.pb.go

```diff
@@ -1,17 +1,18 @@
 // Code generated by protoc-gen-gogo. DO NOT EDIT.
 // source: accountWrapperMock.proto
 
-package mock
+package state
 
 import (
 	fmt "fmt"
-	_ "github.com/gogo/protobuf/gogoproto"
-	proto "github.com/gogo/protobuf/proto"
 	io "io"
 	math "math"
 	math_bits "math/bits"
 	reflect "reflect"
 	strings "strings"
+
+	_ "github.com/gogo/protobuf/gogoproto"
```

Comment:
change `option go_package =` in `accountWrapperMock.proto`

Reply:
done